index.js

```diff
@@ -0,0 +1,67 @@
+console.log("hello, vanilla.");
+
+
+//재시작
+function restart(){
+    input.value="";
+    start();
+    alert("재시작합니다.")
+}
+
+//게임버튼 클릭시 랜덤 숫자 생성
+let start = function(){
+    let numberAll =[0,1,2,3,4,5,6,7,8,9];
+    let ansNum=[];
+    for(let i=0; i<3; i++){
+        let num = numberAll[Math.floor(Math.random()*numberAll.length)];
+        ansNum.push(num);
+    }
+    return ansNum;
+};
+
+let doNum = start();
+console.log(doNum);
+
+//입력값이 세자리 숫자가 아닌 경우 경고창 띄워주기
+let form = document.querySelector("form"),
+    input = document.querySelector("input");
+let intNum =function (){
```

Comment:
함수명은 대부분 동사로 시작합니다. 입력한 값의 유효성을 확인하는 함수이기 때문에 저였으면 `validationNumber`정도의 명확한 함수명을 작성했을 것 같습니다 :D

또한 아래와 같이 함수 표현식을 작성하면 좋을 것 같습니다.
```suggestion
const validationNumber = () => {
```